lib/BC/SleighLifter.cpp

```diff
@@ -700,24 +736,65 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
   static std::unordered_set<OpCode> INTEGER_COMP_OPS;
 
 
+  struct VisitExit {
+    PcodeToLLVMEmitIntoBlock &th;
```

Comment:
I am ok with short names in use-cases like this, but as a random reader I cannot deduce what `th` is, so maybe chose something else?

Reply:
[71a4ddf](https://github.com/lifting-bits/remill/pull/645/commits/71a4ddf6d96b25b0b09ba908ad4b91fb82624e34)